src/Qowaiv/Date.cs

```diff
@@ -370,7 +368,19 @@ public Date Add(DateSpan value, bool daysFirst)
     [Pure]
     public Date AddMilliseconds(double value) => new(m_Value.AddMilliseconds(value));
 
-    #endregion
+    /// <summary>Returns true if the date is in the specified month, otherwise false.</summary>
+    /// <param name="month">
+    /// The <see cref="Qowaiv.Month"/> the date should be in.
+    /// </param>
+    [Pure]
+    public bool In(Month month) => !month.IsEmptyOrUnknown() && Month == (int)month;
```

Comment:
I prefer IsIn(...) to emphasize that it is a boolean check.

Reply:
That is a good suggestion!